errors/truncate_error.go

```diff
@@ -0,0 +1,71 @@
+package errors
+
+import (
+	"github.com/cashapp/blip"
+)
+
+type TruncateErrorPolicy struct {
+	Policy
```

Comment:
Don't embed; avoid embedding in general because it hides which method belongs to which struct, which is a bit of magic, and Go doesn't like magic. E.g. it didn't occur to me at first that `p.Metric` comes from `Policy`, not `TruncateErrorPolicy`--and I wrote `Policy` (but, granted, that was awhile ago).